source/_components/sensor.sql.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "SQL Sensor"
+description: "Instructions how to integrate SQL sensors into Home Assistant."
+date: 2018-02-03 00:22
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sql.png
+ha_category: Sensor
+ha_release: 0.64
+---
+
+
+The `SQL` sensor platform enables you to use values from an [SQL](https://en.wikipedia.org/wiki/SQL) database supported by the [sqlalchemy](https://www.sqlalchemy.org) library, to populate a sensor state (and attributes).
+This can be use to present statistic about home_assistant sensors if used with the recorder component database. It can also be used with an external data source.
```

Comment:
This can be use to present statistic about Home Assistant sensors if used with the recorder.

Reply:
Got the type on home_assistant, but I would be careful with "if used with the recorder." since this sensor does not depend on the recorder component.